packages/client/src/components/BigMovieCard/BigMovieCard.jsx

```diff
@@ -111,14 +126,36 @@ export const BigMovieCard = ({ currentMovie }) => {
             alignSelf: 'flex-end',
           }}
         >
+          {user ? (
```

Comment:
```suggestion
          {user  && (
```
then remove the ""

Reply:
Thanks for the suggestion, it looks more concise now!